browser/src/canvas/sections/ScrollSection.ts

```diff
@@ -105,6 +105,7 @@ export class ScrollSection extends CanvasSectionObject {
 		this.sectionProperties.animatingScroll = false;
 
 		this.sectionProperties.animateWheelScroll = (<any>window).mode.isDesktop();
+		this.sectionProperties.lastElapsedTime = 0;
```

Comment:
I think we need to reset this also in `onAnimationEnded`?

Reply:
This is always reset when starting the scroll animation and only used by the scroll animation. We could have it in `onAnimationEnded` instead, but it feels more logical to group it with the start and this way also allows us to see the lastElapsedTime after an animation is finished (not that we need to currently, or likely ever...)